rerun_py/rerun_sdk/rerun/datatypes/angle.py

```diff
@@ -24,14 +24,25 @@ class Angle(AngleExt):
 
     inner: float = field(converter=float)
     """
-    Radians (float):
+    Must be one of:
+
+    * Radians (float):
         3D rotation angle in radians. Only one of `degrees` or `radians` should be set.
 
-    Degrees (float):
+    * Degrees (float):
         3D rotation angle in degrees. Only one of `degrees` or `radians` should be set.
     """
 
     kind: Literal["radians", "degrees"] = field(default="radians")
+    """
+    Possible values:
+
+    * "Radians":
+        3D rotation angle in radians. Only one of `degrees` or `radians` should be set.
+
+    * "Degrees":
+        3D rotation angle in degrees. Only one of `degrees` or `radians` should be set.
```

Comment:
Did you check how this looks in the generated python docs?

Reply:
Yep, that's the reason I leave so much space because otherwise it doesn't look great. This is what it looks like with the changes from this PR:

<img width="912" alt="image" src="https://github.com/rerun-io/rerun/assets/1220815/6d5b468c-e787-4de8-884f-fa90211ae4f4">

Reply:
hmm actually could still be better. the "only one of should be set" is a bit confusing ion this case, but then I also find the entire pattern a bit confusing 🤷